src/NuGet.Core/NuGet.Commands/RestoreCommand/Logging/WarningPropertiesCollection.cs

```diff
@@ -160,6 +160,7 @@ public static void ApplyProjectWideWarningsAsErrorProperties(ILogMessage message
                 {
                     // If the project wide AllWarningsAsErrors is true and the message has a valid code or
                     // Project wide WarningsAsErrors contains the message code then upgrade to error.
+                    message.Message = "(WarningsAsErrors) " + message.Message;
```

Comment:
I feel like this is a needed change, but something about having the note in front of the message just feels weird. 

Unfortunately I don't really have any better suggestions. 

Curious what other components owners think about this change.

Reply:
How about `(WarningsAsErrorsSetting)` to give more intuition?

Reply:
> WarningAsError NU1605: The normal message

You have to remember what VS does in the error lsit.